lambda_code/src/models/utils.py

```diff
@@ -39,13 +42,39 @@ def generic_string_validation(
                 + str(", ".join(predefined_values))
             )
 
+    if is_postal_code:
```

Comment:
Spaces are allowed (in fact must exist according to the spec for the only postcode field) for postcode, so that would not be a sensible combination of arguments and it's correct that it should throw